cli/flags.rs

```diff
@@ -21,11 +21,6 @@ pub struct DenoFlags {
   pub allow_run: bool,
   pub allow_high_precision: bool,
   pub no_prompts: bool,
-  pub types: bool,
-  pub prefetch: bool,
-  pub info: bool,
-  pub fmt: bool,
-  pub eval: bool,
 }
 
 impl<'a> From<ArgMatches<'a>> for DenoFlags {
```

Comment:
Can this `DenoFlags::<ArgMatches>::from` implementation be combined into `set_flags`? I feel like parsing is now being broken up into two parts.